proto/greptime/v1/ddl.proto

```diff
@@ -148,14 +148,10 @@ message DropColumns { repeated DropColumn drop_columns = 1; }
 message ChangeColumnTypes { repeated ChangeColumnType change_column_types = 1; }
 
 message ChangeFulltext {
-    message FulltextOptions {
-        bool enable = 1;
-        string analyzer = 2;
-        bool case_sensitive = 3;
-    }
-
     string column_name = 1;
-    FulltextOptions options = 2;
+    bool enable = 2;
+    string analyzer = 3;
```

Comment:
nit: Using a enum, it can simplify struct conversion.

```
enum Analyzer {
  ENGLISH = 0;
  CHINIESE = 1;
}
```

```
message ChangeFulltext {
    ...
    Analyzer analyzer = 3;
    ...
}
```